generator/generator.go

```diff
@@ -275,8 +275,8 @@ func (g *generator) Build(ctx *builder.MethodContext, sourceID *xtype.JenID, sou
 			Name:          name,
 			Source:        xtype.TypeOf(source.T),
 			Target:        xtype.TypeOf(target.T),
-			Mapping:       map[string]string{},
-			IgnoredFields: map[string]struct{}{},
```

Comment:
Thanks for your contribution, sadly this has side effects, as the ignore will now be passed to every nested method member. F.ex, in this test, both Name properties from OutputOne and OutputTwo will be ignored, but only the first should be.

I'll think about a solution without side effects.
```go
package structs

// goverter:converter
type Converter interface {
    // goverter:ignore Name
    Convert(source *InputOne) *OutputOne
}

type InputOne struct {
    Name string
    Age int
    Nested InputTwo
}
type InputTwo struct {
    Age int
}
type OutputOne struct {
    Name string
    Age int
    Nested OutputTwo
}
type OutputTwo struct {
    Name string
    Age int
}

```